lib/include/hse_ikvdb/c0.h

```diff
@@ -1,14 +1,15 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 /*
- * Copyright (C) 2015-2020 Micron Technology, Inc.  All rights reserved.
+ * Copyright (C) 2015-2021 Micron Technology, Inc.  All rights reserved.
```

Comment:
To be fair when I wrote this PR the year was still 2021! Will update it.

Reply:
> One gnu-ism that will be hard to get rid of is nested functions. @nabeelmmd maybe if you get a chance, you could take a look at nested functions that we currently have in the code since at least one of them is in mpool :).

Sure, will check this out.